tests/actions/IOUTest.ts

```diff
@@ -508,6 +508,7 @@ describe('actions/IOU', () => {
                 selfDMReport.reportID,
                 CONST.IOU.ACTION.CATEGORIZE,
                 reportActionableTrackExpense?.reportActionID,
+                {choice: CONST.ONBOARDING_CHOICES.MANAGE_TEAM},
```

Comment:
Is this the only test that needs to be updated? What about the parameter changes for the workspace creation methods?